conans/test/functional/command/inspect_test.py

```diff
@@ -438,3 +438,12 @@ class Lib(ConanFile):
         client.save({"conanfile.py": conanfile})
         client.run("inspect . --raw=default_options")
         self.assertEqual("dict=True\nlist=False", client.out)
+
+    def test_initial_inspect_without_registry_test(self):
+        client = TestClient()
+        client.save({"conanfile.py": self.conanfile})
+        client.run("export . user/channel")
+        os.remove(client.cache.remotes_path)
+        client.run("inspect MyPkg/1.2.3@user/channel --raw=version")
+        self.assertIn("WARN: Remotes registry file missing, creating default one", client.out)
```

Comment:
the test should be that there is NO warning.

Maybe the ``TestClient`` doesn't have this error, but it only happens with the real client app.

Reply:
Sure!!